public/xmainWindow.js

```diff
@@ -8,7 +8,7 @@ const express = require('express');
 const {router} = require('../src/main/xserver/xroutes/xroutes.js');
 const exApp = express();
 const server = require('http').Server(exApp);
-// require('electron-reload')(__dirname);
+require('electron-reload')(__dirname);
```

Comment:
Is this used? If not, please remove or return to being commented.

Reply:
so Will comment this line because he could not work effectively with electron hot reload but I uncommented it to get it back in sin it was not an issue for me